pkg/irreparabledb/dbx/irreparabledb.dbx

```diff
@@ -0,0 +1,24 @@
+// dbx.v1 golang irreparabledb.dbx .
+model irreparabledb (
+	key segmentpath
+
+	field segmentpath blob
+	field segmentdetail blob (updatable)
+	field pieces_lost_count int64 (updatable)
+	field seg_damaged_unix_sec int64 (updatable)
+	field repair_attempt_count int64 (updatable)
+)
+
+create irreparabledb ( )
+update irreparabledb ( where irreparabledb.segmentpath = ? )
+delete irreparabledb ( where irreparabledb.segmentpath = ? )
+read one (
+  select irreparabledb 
+  where  irreparabledb.segmentpath = ?
+)
+read limitoffset (
```

Comment:
Is this used some where?

Reply:
It is currently not used. Removed the unused code.